client/app/scripts/superdesk-archive/related-item-widget/configuration.html

```diff
@@ -0,0 +1,35 @@
+<div class="widget-config">
```

Comment:
I don't think we need configuration for this widget, as it will not be used in dashboard.